homeassistant/components/media_player/enigma.py

```diff
@@ -0,0 +1,164 @@
+"""
+homeassistant.components.media_player.enigma
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Beta implementation of the Enigma2 media player.
+Currently only shows whats playing on the box.
+
+You should have a recent version of OpenWebIf installed
+on your E2 box.
+
+There is no support for username/password authentication
+at this time.
+
+Configuration:
+
+To use the media_player you will need to add something like the
+following to your config/configuration.yaml
+
+media_player:
+    platform: enigma
+    name: Vu Duo2
+    host: 192.168.1.26
+    port: 80
+
+Variables:
+
+host
+*Required
+This is the IP address of your Enigma2 box. Example: 192.168.1.32
+
+port
+*Optional
+The port your Enigma2 box uses, defaults to 80. Example: 8080
+
+name
+*Optional
+The name to use when displaying this Enigma2 switch instance.
+
+"""
+from homeassistant.components.media_player import (
+    MediaPlayerDevice, STATE_NO_APP, ATTR_MEDIA_STATE,
+    ATTR_MEDIA_CONTENT_ID, ATTR_MEDIA_TITLE, ATTR_MEDIA_DURATION,
+    ATTR_MEDIA_VOLUME, MEDIA_STATE_PLAYING, MEDIA_STATE_STOPPED)
+
+import logging
+import openwebif.api
```

Comment:
Could you have the import follow the same pattern as for the Chromecast:

1. [Catch exception if import fails](https://github.com/balloob/home-assistant/blob/dev/homeassistant/components/media_player/cast.py#L11)
2. [Raise appropriate error during setup](https://github.com/balloob/home-assistant/blob/dev/homeassistant/components/media_player/cast.py#L33)